master/buildbot/reporters/gerrit.py

```diff
@@ -203,6 +204,7 @@ def __init__(self, func):
             self.gerrit_version = None
 
         def outReceived(self, data):
+            data = bytes2NativeString(data)
```

Comment:
This is used for `gerrit_version = LooseVersion(data)` which expects a str, not a bytes-like object;

> TypeError: cannot use a string pattern on a bytes-like object